content/en/docs/architecture/step-registry.md

```diff
@@ -959,3 +959,59 @@ tests:
   steps:
     workflow: openshift-e2e-disruptive
 {{< / highlight >}}
+
+## VPN connection
```

Comment:
I feel like this section should go to https://deploy-preview-249--zealous-shirley-0f56b3.netlify.app/docs/how-tos/adding-a-cluster-profile/

Reply:
I like that idea.  I'll leave the (very short) test-specific part here, move the cluster profile setup there, and cross-reference them.

Reply:
Done, new links in the PR description.